libs/utils/utils.js

```diff
@@ -461,6 +461,16 @@ export async function loadBlock(block) {
         await init(block);
       } catch (err) {
         console.log(`Failed loading ${name}`, err);
+
+        const { message, errorType, module } = error;
+        if (message && errorType && module) {
+          window.lana.log(`${message}; blockPath: ${blockPath} | href: ${window.location.href} | ${err.reason || err.error || err.message || err}`, {
+            clientId: 'feds-milo',
```

Comment:
This file is used by all Milo consumers, we should not add any logic here that is not relevant to everyone, like this FEDS-specific client id.